### PR TITLE
can unselect purchasable item

### PR DESCRIPTION
**Pull request by jromero86**

### CODE REVIEW

File: src/Elcodi/Component/Cart/Entity/Traits/PurchasableWrapperTrait.php [hunk 1]
```diff
@@ -132,7 +139,7 @@ public function getVariant()
     /**
      * Sets quantity
      *
-     * @param integer $quantity Quantity
+     * @param int $quantity Quantity
```
Comment (mmoreram) [inline src/Elcodi/Component/Cart/Entity/Traits/PurchasableWrapperTrait.php]: Why?

File: src/Elcodi/Component/Cart/Entity/Traits/PurchasableWrapperTrait.php [hunk 2]
```diff
@@ -55,8 +55,15 @@
      *
      * @return $this Self object
      */
-    public function setPurchasable(PurchasableInterface $purchasable)
+    public function setPurchasable(PurchasableInterface $purchasable = null)
     {
+        //unselect purchasable
+        if(!$purchasable)
```
Comment (mmoreram) [inline src/Elcodi/Component/Cart/Entity/Traits/PurchasableWrapperTrait.php]: This should complain the code style

**Comment by mmoreram** [comment]
This change is so strange. Could you please add some description of the why of this change please?

Thanks.

**Comment by mmoreram** [comment]
And could you please rebase all these commits in one single commit?

**Comment by javiereguiluz** [comment]
@mmoreram you meant *squash* commits, not *rebase* them, right?

**Comment by mmoreram** [comment]
@javiereguiluz Well, you can *squash* them using rebase interactive ;)

**Comment by rogergros** [comment]
@jromero86 The commits had been moved to https://github.com/elcodi/elcodi/pull/641 ?

**Comment by jromero86** [comment]
Yes. Thanks :)

**Pull request closed without merging**